### DISCUSSION TITLE
Custom Tailwind utility not updating with CSS Modules

### DISCUSSION THREAD

**Discussion by FabienMotte** [vercel/next.js#27892] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

14.17.1

### What browser are you using?

Chrome v92.0.4515.107

### What operating system are you using?

macOS 11.5.1

### How are you deploying your application?

Netlify

### Describe the Bug

Hi there! 👋

I'm having an issue using a Tailwind custom utility with CSS Modules in a Next.js app.
Changes made to the custom utility aren't reflected in the browser in development mode.
I need to restart the development server for the changes to appear.

### Expected Behavior

The changes should be reflected in the browser with Fast Refresh.

### To Reproduce

**Reproduction repository:** https://github.com/FabienMotte/tailwind-nextjs-css-modules/

Steps to reproduce:
1. Clone the reproduction repo, `npm i`, start the Next.js server with `npm run dev`
2. Create a custom utility in layer `utilities` (https://github.com/FabienMotte/tailwind-nextjs-css-modules/blob/main/styles/index.css#L6)
3. Use that custom utility in a CSS Modules file (https://github.com/FabienMotte/tailwind-nextjs-css-modules/blob/main/components/Test.module.css#L2)
4. Make a change to the custom utility : `text-green-500` -> `text-red-500` (https://github.com/FabienMotte/tailwind-nextjs-css-modules/blob/main/styles/index.css#L7)
5. The change is not reflected on the test component using CSS Modules.  
If you then restart the Next.js server, the change will happen, but only once.

Thank you!

**Comment by timneutkens** [comment]
This is related to Tailwind not marking `styles/index.css` as a dependency of `components/Test.module.css`. Can you report this on the Tailwind repository?

**Reply by FabienMotte** [reply]
Thanks for your answer! The issue is reported here: https://github.com/tailwindlabs/tailwindcss/issues/5126
But I'm still waiting for answers
